pkg/operation/botanist/component/istio/istiod_test.go

```diff
@@ -1660,8 +1669,7 @@ spec:
         
         service.istio.io/canonical-name: "istio-ingressgateway"
         service.istio.io/canonical-revision: "1.7"
-        networking.gardener.cloud/to-dns: allowed
-        networking.resources.gardener.cloud/to-all-shoots-vpn-seed-server-tcp-1194: allowed
+        networking.gardener.cloud/to-dns: allowed` + additionalAnnotations + `
```

Comment:
Small nit: Wouldn't `additionalLabels` be a better name?

Reply:
Yes, my bad 😄 Thanks for the hint.